Kentico.Kontent.AspNetCore/Webhooks/Models/Common.cs

```diff
@@ -32,6 +18,12 @@ public class Message
         [JsonPropertyName("id")]
         public Guid Id { get; set; }
 
+        /// <summary>
+        /// Identifier of a Kentico Kontent project.
```

Comment:
For the new code, we would like to use just "Kontent".

Reply:
Updated.